packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -31,23 +35,40 @@ let {
   ariaInvalid?: boolean | 'grammar' | 'spelling';
   ariaLabel?: string;
   children?: Snippet;
+  multiple?: boolean;
+  label?: string;
 } = $props();
 
 let opened: boolean = $state(false);
 let selectLabel: string = $state('');
+let selectLabelPlus: string | undefined = $state(undefined);
 let highlightIndex: number = $state(-1);
 let comp: HTMLElement;
 
 const pageStep: number = 10;
 
 onMount(() => {
-  if (!value && options?.length > 0) {
+  if (!multiple && !value && options?.length > 0) {
     value = options[0].value;
```

Comment:
This is a real problem that we cannot have undefined as a valid option, like nothing selected